include/crypto/evp.h

```diff
@@ -565,6 +565,11 @@ struct evp_pkey_st {
      * a copy of that key's dirty count.
      */
     size_t dirty_cnt_copy;
+
+    /* Cache of domain parameter / key information */
+    int bits;
+    int security_bits;
+    int size;
```

Comment:
This is a sidenote, not a request for changing anything, but to discuss if you believe there could be value in an alternative.

@levitte what do you think about including these in a `struct`?

```C
struct {
    int bits;
    int security_bits;
    int size;
} cache;
```

I believe it could be a good idea to embed the fact that they are under the control of  `evp_keymgmt_cache_pkey()` and help future contributors be aware of it.

Reply:
Sure

Reply:
Done